addons/website/static/src/js/widgets/theme.js

```diff
@@ -325,7 +325,7 @@ var ThemeCustomizeDialog = Dialog.extend({
                         var xmlid = $item.data('xmlid');
 
                         var renderingOptions = _.extend({
-                            string: $item.attr('string') || data.names[xmlid.split(',')[0].trim()],
+                            string: $item.attr('string') || $item.data('variable') || data.names[xmlid.split(',')[0].trim()],
```

Comment:
Why would you show a variable name to the user ? x)

Reply:
I'll ask you about that tomorrow because I have an error I don't figure it out :) except if the solution now proposed in my new push is ok